devito/mpi/routines.py

```diff
@@ -765,9 +745,15 @@ def _make_remainder(self, hs, key, callcompute, region):
 
         dim = Dimension(name='i')
         regioni = IndexedPointer(region, dim)
```

Comment:
region_i?

Reply:
yes absolutely. Thought I had fixed it. Missed it actually. Will fix shortly